docker/Dockerfile

```diff
@@ -9,13 +9,13 @@ RUN JAVA_TOOL_OPTIONS="-Djdk.lang.Process.launchMechanism=vfork" "$JAVA_HOME/bin
          --compress=2 \
          --output /javaruntime
 
-FROM ubuntu:20.04
+FROM ubuntu:21.10
```

Comment:
why use ubuntu:latest like we are in web3signer?

Reply:
latest matches to `20.04` or whatever LTE is currently pointing to. `21.10` as suggested by synk bot is not an LTE, hence removed `latest` and mentioned `21.10` instead.